tfhe/src/high_level_api/keys/client.rs

```diff
@@ -112,7 +104,7 @@ pub trait RefKeyFromKeyChain: Sized {
 /// - The identifier (or identifier chain) that points to the member in the `ClientKey` that holds
 ///   the key for which the trait is implemented.
 /// - Type Variant used to identify the type at runtime (see `error.rs`)
-#[cfg(any(feature = "integer", feature = "shortint", feature = "boolean"))]
```

Comment:
Why remove integer here?

Reply:
This macro is not used in the integer part of the hlapi